velox/docs/monthly-updates/november-2023.rst

```diff
@@ -0,0 +1,96 @@
+********************
+November 2023 Update
+********************
+
+Documentation
+=============
+* Add documentation for ``num_cached_file_handles`` :doc:`configuration property</configs>`.
+
+
+Core Library
+============
+* Add support to export constant vector as Arrow REE array. :pr:`7327`, :pr:`7398`
+* Add `libcpr` a lightweight http client. :pr:`7385`
+* Add spilling support for distinct aggregation. :pr:`7305`
+* Add DirectBufferedInput: a selective BufferedInput without caching. :pr:`7217`
+* Add config ``merge_exchange.max_buffer_size`` to limit the total memory used by exchange clients. :pr:`7410`
+* Add configs ``sort_writer_max_output_rows`` and ``sort_writer_max_output_bytes`` to limit memory usage of sort writer. :pr:`7339`
+* Add support for TIMESTAMP type in Arrow bridge. :pr:`7435`
+* Add termination time to TaskStats. :pr:`7479`
+* Add Presto Type Parser based on Flex and Bison. :pr:`7568`
+* Add PrestoQueryRunner that can be used to verify test results against Presto. :pr:`7628`
+* Add support for named row fields in type signature and binding. :pr:`7523`
+* Add support for spilling aggregations over sorted inputs. :pr:`7526`
+* Add support to shrink cache if clients such as Prestissimo detect high memory usage on a worker. :pr:`7547`, :pr:`7645`
+* Add support to lazily create the spill directory. :pr:`7660`
+* Add ``RowVector::childAt(name)`` API. :pr:`7737`
+* Add DuckDB version 0.8.1 as an external dependency and remove DuckDB amalgamation. :pr:`6725`
+* Fix distinct aggregations with global grouping sets to return a row per set with the group id on empty input. :pr:`7353`
+* Fix incorrect runtime stats reporting when memory arbitration is triggered. :pr:`7394`
+* Fix Arrow bridge to ensure the null_count is always set and add support for null constants. :pr:`7411`
+* Fix ``Timestamp::toMillis()`` to overflow only if the final result overflows. :pr:`7506`
+* Fix DECIMAL to VARCHAR cast by adding trailing zeros when the value is `0`. :pr:`7588`
+* Fix producing wrong result from aggregations over sorted inputs with spilling. :pr:`7764`
```

Comment:
Remove as we are announcing spill added here, not need to say added Xxx, fixed Xxx